server/neptune/workflows/internal/deploy/revision/queue/worker.go

```diff
@@ -102,14 +105,28 @@ func (w *Worker) work(ctx workflow.Context, revision string) error {
 	logger.Info(ctx, fmt.Sprintf("latest deployed revision %s", deployedRevision))
 
 	// TODO: fill in the rest
-
+	// Spin up a child workflow to handle Terraform operations
+	childWorkflowOptions := workflow.ChildWorkflowOptions{
+		TaskQueue: terraform.TaskQueue,
+		//TODO: match workflow id to format used by deploy workflow
+		WorkflowID: fmt.Sprintf("%s-%s", w.Repo.GetFullName(), w.Root.Name),
```

Comment:
should we just use the deploy id that's generated here?

Reply:
I wanted there to be metadata in our workflow id to make it easily recognizable when debugging, observing its process via UI, etc. What do you think?

Reply:
I'd prefer to use an obtuse identifier so its easy to like correlate things from the UI/DB/GH UI with one identifier.  debugging is made easier by using search attributes: https://docs.temporal.io/visibility/ which we should be able to leverage for metadata.